test/docker-entrypoint.sh

```diff
@@ -5,19 +5,19 @@
 # Credits: https://github.com/thecodingmachine/docker-images-php.
 
 set +e
-mkdir testing_file_system_rights.foo
+mkdir -p testing_file_system_rights.foo
 chmod 700 testing_file_system_rights.foo
 su gotenberg -c "touch testing_file_system_rights.foo/foo > /dev/null 2>&1"
 HAS_CONSISTENT_RIGHTS=$?
 
 if [[ "$HAS_CONSISTENT_RIGHTS" != "0" ]]; then
     # If not specified, the DOCKER_USER is the owner of the current working directory (heuristic!).
-    DOCKER_USER=$(stat -c '%U' "$(pwd)")
+    DOCKER_USER=$(stat -c '%u' "$(pwd)")
```

Comment:
Note that I made these changes in `docker-entrypoint.sh` since I kept having the problem where `DOCKER_USER` was set to `UNKNOWN`, causing an error and preventing the test container from running. What I did here is switch from getting the user name to getting the user id. This should be more reliable hopefully.